firmware/baseband/proc_capture.cpp

```diff
@@ -102,10 +102,12 @@ void CaptureProcessor::sample_rate_config(const SampleRateConfigMessage& message
     // waterfall runs slower. Reduce the update interval so it runs faster.
     // NB: Trade off: looks nicer, but more frequent updates == more CPU.
     if (sample_rate >= 1'500'000)
-        spectrum_interval_samples /= (sample_rate / 500'000);
+        spectrum_interval_samples /= (sample_rate / 750'000);
 
-    // Mystery scalars for decimator configuration.
-    // TODO: figure these out and add a real comment.
+    // Those two below "mystery" scalars are widely used in many App's, for decimator (decim0, decim1)  configuration.
```

Comment:
This comment doesn’t really explain why. Also, given they are all the same and we don’t know exactly why these values, we should probably move the constants into function defaults instead of having these magic numbers all over the place.

Reply:
Well , I tried to add my  best current understanding and concept hints about those two constants .

I think  we are giving more flexiblity leaving those two scalars outside , then in every application, the developper could optimize each needed value.  (depending on the selected Image Reject filter, or bandwith , you may need to make final adjustment ) .

Now , we could see that many App's are just copy / paste those values, because most likely it will work well. 
But strickly speaking , I am assuming that the original designer , make the related signal path level diagramm,  and assumed a proper dynamic margin , to make that convertion .

And as here we have the waterfall spectrum , in fact we can see that those values are well optimized.

As example , just for more visual understanding , I just modified the first constant , 
Using original  (decim_0_scale = 0x2000000;) -  we have good usefull balanced dynamic margin  and   no saturation .
![image](https://github.com/eried/portapack-mayhem/assets/86470699/3c81b3f5-ff11-49ea-9fd2-c51340479e96)


Using  (decim_0_scale = 0x20000, intentionally reduced ;) 
-  we have very poor signal  dynamic margin  , but obviously  no saturation .(but not well optimized ) 
![image](https://github.com/eried/portapack-mayhem/assets/86470699/d826d289-c284-4967-9cf8-716f9ab11a13)


 Of course that dark picutre can be boosted , increasing LNA and GAIN , but you will have less  dynamic signal margin if those constant are not well selected.   
My  intention was  to add  the background of those magic numbers, not discuss why var- parameters and not integrated fixed constant .  I still think that better leave them as a config var parameters.

Reply:
I’m not saying they should not be configurable. They should have parameter defaults if everything is just using the same copy pasted value.

Reply:
I'm going to move those constants to named values and add configure defaults. Revert you comment changes and we might be able to avoid a merge conflict. Otherwise, this is ok.